pkg/vm/vm.go

```diff
@@ -1313,3 +1326,9 @@ func validateMapKey(key *Element) {
 		panic("key can't be a collection")
 	}
 }
+
+func (v *VM) checkInvocationStack() {
```

Comment:
Maybe it's worth renaming it to `checkInvocationStackSize`? Otherwise it's not obvious what do we check there.

Reply:
Yes, it is more clear.